src/compose/ComposeBox.js

```diff
@@ -320,14 +322,18 @@ class ComposeBox extends PureComponent<Props, State> {
   };
 
   handleSend = () => {
-    const { dispatch, narrow, caughtUp, _ } = this.props;
+    const { dispatch, narrow, caughtUp, _, isTopicMandatory } = this.props;
     const { message } = this.state;
+    const topic = this.state.topic;
 
     if (!caughtUp.newer) {
       showErrorAlert(_('Failed to send message'));
       return;
     }
-
+    if (isTopicMandatory && (topic === '(no topic)' || !topic)) {
+      showErrorAlert(_('Please specify a topic'));
```

Comment:
We should this new user-facing string to static/translations/messages_en.json, in the commit that adds it, so it gets localized; see our [translations doc](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/translations.md).